openghg/standardise/surface/_npl.py

```diff
@@ -54,14 +53,7 @@ def parse_npl(
     site_data = get_site_info()
     site_info = site_data[site_upper][network_upper]
 
-    # mypy doesn't like NaT or NaNs - look into this
-    def parser(date: str):  # type: ignore
```

Comment:
A nice thing about `date_format` is that it uses `NaT` instead of `NaN`, so we don't need to have a parser like this.